test/file.jl

```diff
@@ -382,7 +382,7 @@ if Sys.iswindows()
 else
     @test filesize(dir) > 0
 end
-nowtime = time()
+nowtime = time_ns()
```

Comment:
This doesn't look correct, since this needs to be in the same units as file `mtime`.